chirpstack/src/api/internal.rs

```diff
@@ -496,6 +496,14 @@ impl InternalService for Internal {
             }
         };
 
+        // fetch user again because the provisioning callback url may have updated the user.
+        u = match user::get(&u.id).await {
```

Comment:
I think something like:

```rust
u = user::get(&u.id).await.map_err(|e| e.status())?;
```

Conversation 0
Commits 1
Checks 0

is a bit cleaner. I don't think there is a need to use `match` for this.

Reply:
Done.